src/components/IconClose/IconClose.js

```diff
@@ -5,14 +5,17 @@ import classNames from 'classnames';
 import css from './IconClose.css';
 
 const IconClose = props => {
-  const { className, rootClassName } = props;
+  const { className, rootClassName, size } = props;
   const classes = classNames(rootClassName || css.root, className);
 
+  // Use given size as width and height or default to 12px
+  const svgSize = size ? size : 12;
```

Comment:
There's a bit of catch when sizing SVG icons with width and height. They render graphics (paths, rects, circles etc.) to their own inner viewbox (or should I say coordinate system). 

There's one problem with this concept related to line graphics: if the width of the icon is scaled down, also the line width is scaled down accordingly. So, it is unlikely that the width of the line matches with the width our designer intended it to be. (This is the reason why we have different SVG graphics for different sizes in [IconArrowHead.js](https://github.com/sharetribe/flex-template-web/blob/master/src/components/IconArrowHead/IconArrowHead.js)